zrml/prediction-markets/Cargo.toml

```diff
@@ -48,7 +48,7 @@ mock = [
     "pallet-randomness-collective-flip/default",
     "pallet-timestamp/default",
     "pallet-treasury/default",
-    "serde",
+    "serde/default",
```

Comment:
`std` ?

```suggestion
    "serde/std",
```

Reply:
This is expected behavior, as every dependency in the `mock` feature is loaded with default features.